compiler/tools/fpp-to-cpp/test/component/ActiveStateMachines_S1.cpp

```diff
@@ -0,0 +1,100 @@
+
+// ======================================================================
+// \title  ActiveStateMachines_S1.cpp
+// \author Auto-generated
+// \brief  cpp file for state machine ActiveStateMachines_S1
+//
+// ======================================================================            
+
+#include "ActiveStateMachines_S1.hpp"
```

Comment:
@garthwatney Can you make sure that STARS generates these header includes? Before I saw dependencies on C stdio and C assert. We shouldn't have those dependencies in flight code.

Reply:
Done